scripts/socketplane.sh

```diff
@@ -316,6 +320,12 @@ start_socketplane() {
 
     mkdir -p /var/run/socketplane
     echo $cid > /var/run/socketplane/cid
+
+    if [ "$ps" = "yes" ]; then
+	pscid=$(docker run -d --name powerstrip -v /var/run/docker.sock:/var/run/docker.sock \
+	    -v /etc/socketplane/adapters.yml:/etc/powerstrip/adapters.yml --net=host  \
```

Comment:
Hmm.. I see your point. But if we want to make it seamless we need make an assumption on where certain pieces go. Where is the socketplane script placed in a non-vagrant setup?
We can just say that place the repo .yml file in /etc/socketplane/adapters.yml